libraries/libliftoff/libliftoff.info

```diff
@@ -2,7 +2,7 @@ PRGNAM="libliftoff"
 VERSION="0.4.1"
 HOMEPAGE="https://gitlab.freedesktop.org/emersion/libliftoff"
 DOWNLOAD="https://gitlab.freedesktop.org/emersion/libliftoff/-/archive/v0.4.1/libliftoff-0.4.1.tar.gz"
```

Comment:
You can also use the "release" link https://gitlab.freedesktop.org/emersion/libliftoff/-/releases/v0.4.1/downloads/libliftoff-0.4.1.tar.gz

That MD5SUM matches the one here already and it results in the same file name with wget or a browser (content-disposition header)

Reply:
```sh
$ curl -L -v https://gitlab.freedesktop.org/emersion/libliftoff/-/releases/v0.4.1/downloads/libliftoff-0.4.1.tar.gz -o /dev/null 2>&1 | grep -i content-disposition
< content-disposition: attachment; filename="libliftoff-0.4.1.tar.gz"; filename*=UTF-8''libliftoff-0.4.1.tar.gz
```